src/py42/sdk/queries/fileevents/filters/event_filter.py

```diff
@@ -28,10 +62,15 @@ class EventType(FileEventFilterStringField):
     _term = u"eventType"
 
     CREATED = u"CREATED"
-    MODIFIED = u"MODIFIED"
+    MODIFIED = u"UPDATED"
```

Comment:
what instance are you using to test this/what led to this change? The environment I am checking uses `MODIFIED` for this:
![Screen Shot 2020-07-27 at 11 39 44 AM](https://user-images.githubusercontent.com/5409649/88568094-0a51d780-cffe-11ea-82be-e648e0e3e92e.png)

Reply:
Now when I check again, I too see it as "MODIFIED", my notes still show it as "UPDATED", not sure how I came across it and added to my notes. I will revert the same.